spec/dummy_controller_without_gems_spec.rb

```diff
@@ -150,4 +147,60 @@
       end
     end
   end
+
+  describe '#index_total_count' do
+    let!(:model_count) { 28 }
```

Comment:
:thinking: we really need to create 28 instances? I prefer to create fewer instances, and change the `limit` param :sunglasses:

Reply:
I did it just because all the tests have the same approach but you're right it's not needed... will change it in next commit 👍